engine/runtime-language-arrow/src/main/java/org/enso/interpreter/arrow/ArrowLanguage.java

```diff
@@ -0,0 +1,48 @@
+package org.enso.interpreter.arrow;
+
+import com.oracle.truffle.api.CallTarget;
+import com.oracle.truffle.api.TruffleLanguage;
+import org.enso.interpreter.arrow.node.ArrowEvalNode;
+
+/** An internal language that implements Arrow specification. */
+@TruffleLanguage.Registration(
+    id = ArrowLanguage.ID,
+    name = "Truffle implementation of Arrow",
+    characterMimeTypes = {ArrowLanguage.MIME},
+    defaultMimeType = ArrowLanguage.MIME,
```

Comment:
You have to explicitly specify `internal = true`. By default `internal = false`.